crates/sdk/src/connector/example.rs

```diff
@@ -112,29 +112,3 @@ impl Connector for Example {
         todo!()
     }
 }
-
-#[cfg(test)]
-mod tests {
-    use std::path::PathBuf;
-
-    use axum_test_helper::TestClient;
-    use http::StatusCode;
-
-    use super::*;
-
-    #[tokio::test]
-    async fn capabilities_match_ndc_spec_version() -> Result<()> {
```

Comment:
This test is pointless, now that `ndc_models` exports `VERSION` and we use it in the SDK.